### PR TITLE
Images slider drawer

### PR DESCRIPTION
**Pull request by itsmebipul00**
Designed Slider and Images, added badges and avatars which I deleted previously
PS I'll add the code snippet at the end

### CODE REVIEW

File: Components/Slider/Slider.css [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  font-size: 1.5rem;
+}
+
+.slider {
+  margin: 0;
+  -webkit-appearance: none;
+  width: 100%;
+  height: 10px;
+  border-radius: 5px;
+  background: #780206; /* fallback for old browsers */
```
Comment (zoef-yt) [inline Components/Slider/Slider.css:42]: Use var for colour

**Review by zoef-yt — APPROVED** [review]
With change its good to go. Approved.

File: Components/Slider/Slider.css (hunk 1 above)
Comment (itsmebipul00) [reply Components/Slider/Slider.css:42]: thanks for pointing it overlooked it

**Review by archit7448 — APPROVED** [review]
good to merge

**Pull request merged**